src/core/net_db.cc

```diff
@@ -104,7 +104,7 @@ void NetDb::Run() {
     try {
       auto msg =
 	m_Queue.GetNextWithTimeout(
-	     static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 15 sec
+            static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 15 sec
```

Comment:
The git diff is not correct. What you advise is exactly what I changed in the file. Not sure why it is not showing up in diff.